src/main/java/io/github/intellij/dlanguage/codeinsight/dcd/DCDCompletionServer.java

```diff
@@ -166,7 +165,8 @@ GeneralCommandLine buildDcdCommand(@NotNull final String path) {
         });
 
         // try to auto add dub dependencies
-        final DubConfigurationParser dubParser = new DubConfigurationParser(module.getProject(),
+        // FIXME should not use dub to get dependencies
+        /*final DubConfigurationParser dubParser = new DubConfigurationParser(module.getProject(),
```

Comment:
even though this should be changed probably best keeping it working as it is for this PR with a `// todo: ...` comment as commenting out the code will just break dcd completion

Reply:
Solved, and improvements, now we don’t even need dub to get libraries for DCD command line